### DISCUSSION TITLE
Getting a CORS error when trying to access a deployed studio

### DISCUSSION THREAD

**Discussion by miked-known** [sanity-io/sanity#4503] [Sanity Studio Support]
> An error occured
User is missing one of the follow grants sanity.project/read, sanity.organization.projects/read to perform this operation

My user has an administrator role at both the project and the org level. Any ideas here?

**Comment by miked-known** [comment]
A couple of notes:

1. The studio deployed successfully
2. The CORS url is already added to the API settings...

**Comment by miked-known** [comment]
This feels like a permissions issue I'm just not sure what course of action there is to be taken since I'm already an admin at all levels

**Comment by jamesryan-dev** [comment]
+1 getting the same error for my client, but not my developer (main) account
